static/js/read_receipts.js

```diff
@@ -41,7 +41,7 @@ export function show_user_list(message_id) {
                             $t(
                                 {
                                     defaultMessage:
-                                        "This message has been read by {num_of_people, plural, one {1 person} other {# people}}:",
+                                        "{num_of_people, plural, one {This message has been read by {num_of_people} person:} other {This message has been read by {num_of_people} people:}}",
```

Comment:
Can you also make `read` a link to `/help/read-receipts`? We've been wanting to add that information to the page; I think the `z-link` approach should work for that.

Reply:
Done in a separate commit. I hope that this won't break Transifex pluralization UI.